UPGRADE-7.0.md

```diff
@@ -55,6 +55,12 @@ Support for Adobe Flash was removed. We recommend to search the media manager fo
 extensions "flv", "f4v" and "swf") and remove them.
 The media manager will also display where these files are still used; these usages should also be removed.
 
+## Remove module_cms_search Table
+
+The CMS search module was removed. If the project still needs this module, restore it from an earlier Chameleon release.
```

Comment:
"Restoring" means to restore all the below listed classes, translations, ...?

Reply:
Yes. All of these can be added to project code without relying on Chameleon. I expect that this case is *really* rare, so we shouldn't invest time to simplify this - if developers have difficulties in doing this, we can still help.